workers/unity/Assets/Resources.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 82256c4f21075e640a64c98a803fe7a2
+guid: 5bb31678f3981fd43bd427dbe649978a
```

Comment:
What happened here?

Reply:
I'm guessing Unity decided to give it a different guid now that it's in a different project? 

Good callout though, as this should be with the playmode tests